com.vagueentertainment.Framed.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.vagueentertainment.Framed
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: framed
+modules: 
+  
+  - name: framed
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 Framed.x86_64 /app/bin/Framed.x86_64
+      - install -D Framed.pck /app/bin/Framed.pck
+      - cp -r HoneyComb-redistributable /app/bin/HoneyComb-redistributable
+      - install -D -m755 framed.sh /app/bin/framed
+      - desktop-file-edit --remove-key=PrefersNonDefaultGPU $FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=$FLATPAK_ID $FLATPAK_ID.desktop
+      - install -Dm644 $FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 $FLATPAK_ID.appdata.xml /app/share/appdata/$FLATPAK_ID.appdata.xml
+            
+    sources:
+      - type: archive
+        url: https://github.com/VagueEntertainment/Framed/archive/refs/tags/v0.3.tar.gz
+        sha256: "1558912caf5d83347be02a598b59fa13e09ca758c1a35d18a4d407784292e667"
+        
+      - type: archive
+        url: https://github.com/VagueEntertainment/HoneyComb-redistributable/archive/refs/tags/0.1.1.tar.gz
+        sha256: "69177aaa166089ed14a41c1fe673abfcd08d0f57d2bdb62f524043d1e04ed5dc"
+        dest: HoneyComb-redistributable
+      
+      - type: script
+        dest-filename: framed.sh
+        commands:
+          - export APPDATA="$XDG_DATA_HOME"
+          - export XDG_DATA_HOME="$APPDATA"
+          - export HOME="$APPDATA"
+          - export PATH=$PATH:/app/bin/HoneyComb-redistributable/service
+          - /app/bin/Framed.x86_64 --path=/app/bin/
+          
+      - type: file
+        path: icon.svg
+        
+      - type: file
+        path: com.vagueentertainment.Framed.appdata.xml
+        
+      - type: file
+        path: com.vagueentertainment.Framed.desktop
+        
+  - modules/python3-pip.json      
+  - modules/python3-wheel.json
+  - modules/python3-semantic_version.json
+  - modules/python3-setuptools.json
+  - modules/python3-setuptools_rust.json
+  - modules/python3-setuptools_scm.json
+  - modules/python3-bottle.json
+  - modules/python3-requests.json
+  - modules/python3-netifaces.json
+  - modules/python3-cryptography.json
+  - modules/python3-pycrypto.json
+  - modules/python3-pyopenssl.json
+  - modules/python3-wcwidth.json
+  - modules/python3-prettytable.json
+  - modules/python3-steem.json
+  - modules/python3-hivepy.json
+  - modules/python3-websockets.json
+  - modules/python3-asyncio.json
+        
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home:ro
```

Comment:
this needs the --device=dri permission added otherwise you fall back to software 3d rendering and things get very, very slow.